cmd/subnetcmd/delete.go

```diff
@@ -34,10 +34,12 @@ func deleteSubnet(cmd *cobra.Command, args []string) error {
 	}
 
 	if sidecar.VM == models.CustomVM {
-		if _, err := os.Stat(customVMPath); err == nil {
-			// exists
-			os.Remove(customVMPath)
-		} else {
+		if _, err := os.Stat(customVMPath); err != nil {
+			return err
```

Comment:
Like I said in the PR description
```
There are some places where it wasn't clear to me whether to handle a previously unhandled error, or to ignore it.
```
Totally up to you guys to decide what the right thing to do is in cases like this one

Reply:
OK, but the point is that this is now changing the semantics inside your PR. 

So my suggestion here would be to ignore the error if the file does not exist and not return an non-nil `err`, requiring a change in this PR. @dasconnor @felipemadero

Reply:
Sure, do you want me to remove all changes in this PR that deal with the success of file deletions?

Reply:
I think we can approve this if keeping the same sematics. That is not returning the new errors, instead just make a log.

Reply:
Yeah, I think we want to keep the file deletion logic, that was intentional, mostly we don't care if we can't delete something because it's not actually there.